onnxscript/tools/benchmark/export_model.py

```diff
@@ -59,6 +60,7 @@ def main(args=None):
 
     # Import is delayed so that help is being display faster (without having to import heavy packages).
     import onnxscript.tools
+    import onnxscript.tools.memory_peak as mpk
```

Comment:
```suggestion
    import onnxscript.tools.memory_peak
```

doesn’t look like it’s used a lot, so

Reply:
done